common/proto_common.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.2
-// 	protoc        v5.28.2
+// 	protoc-gen-go v1.27.1
+// 	protoc        v3.12.4
```

Comment:
Please update the protoc version, this is two major releases behind and changes the content.